src/main/resources/assets/minecolonies/lang/manual_en_us.json

```diff
@@ -1376,7 +1376,7 @@
   "com.minecolonies.gui.visiblestatus.digging": "Digging a grave",
   "com.minecolonies.gui.visiblestatus.burying": "Burying a citizen",
 
-  "entity.fisherman.messagewatertoofar": "My hut seems to be too far away from water, and I can't find any water to fish in! It should be an area of at least 7x7 blocks, and the same height as my hut.",
+  "entity.fisherman.messagewatertoofar": "I can't find any water to fish in! There should be an area of at least 7x7x2 water blocks nearby.",
```

Comment:
maybe something more verbose, like "7 wide, and 7 long and at least 2 deep" or so

Reply:
Does it no longer need to be the same Y level as the hutblock?

Reply:
no

Reply:
Awesome! Thanks! My fisher design can happen! =D

Reply:
Being at the same y-level wasn't a requirement even in 1.19.  idk when it was actually fixed, but it was long long ago.